src/mqueryfront/src/QueryField.js

```diff
@@ -91,6 +94,37 @@ class QueryField extends Component {
     }
 
     render() {
+        var options = [];
+        var i;
+        for (i = 0; i < this.props.availableTaints.length; i++) {
+            options.push({
+                label: this.props.availableTaints[i],
+                value: this.props.availableTaints[i],
+            });
+        }
+
+        var multiselect = "";
+
+        if (this.props.availableTaints != "") {
```

Comment:
isn't availableTaints a list? why compare with empty string

Reply:
I tried to change this but every time I tried sth different it failed. What is your suggestion (working) ?

Reply:
to be clear, did you try:
```javascript
if (this.props.availableTaints)
```
and it doesn't work?

Reply:
No. If there are no taints multiselect is still displayed but without taints.

Reply:
Empty list is also truthy in JavaScript. You need to check `if(this.props.availableTaints.length)`.

Psst... `[] == []` is also false :D

Reply:
but `[] == ""` is true and that's why current code is working

Reply:
Ok change with "if(this.props.availableTaints.length)" works right now. Tested manually.